### ISSUE TITLE
 error: 'Path' is not a member type of 'Path'

### ISSUE THREAD

**Issue by rickwierenga** [mxcl/Path.swift#55]
I'm not sure if this is my issue, but I get this error when I try to do anything with Path on Google Colab:

```
expression produced error: error: /tmp/expr56-6421f7..swift:1:70: error: 'Path' is not a member type of 'Path'
Swift._DebuggerSupport.stringForPrintObject(Swift.UnsafePointer<Path.Path>(bitPattern: 0x7f6d9bca1340)!.pointee)
                                                                ~~~~ ^

Path.Path:1:36: note: 'Path' declared here
@dynamicMemberLookup public struct Path : Equatable, Hashable, Comparable {
```

My code:
```swift
%install-location $cwd/swift-install
%install '.package(url: "https://github.com/mxcl/Path.swift", from: "0.16.1")' Path
```

```swift
import Path
Path("/")
// or:
Path.root
```

**Comment by bosr** [comment]
Hi @rickwierenga, did you find a workaround?

Just in case, the problem does not come from `Path` but from how the DebuggerSupport reflects. The following works fine:

```swift
let p = Path.home / "s4tf"
print("\(p)")
```

Cheers!

**Comment by rickwierenga** [comment]
Weird. [Here](https://colab.research.google.com/drive/1V-Wm3DrEFBwkfIdGWfsBSxYazoHOVA22) is a Colab test and I can confirm your code works perfectly fine... I have no clue as to why this happens. No, I have not found a workaround yet.

*only tested this on Colab*

**Comment by mxcl** [comment]
Possibly a Swift compiler bug. The diagnostic certainly looks more fundamental than a source error.

**Comment by mxcl** [comment]
I added a `PathStruct` `typealias` since there is no fix or workaround with Swift 5.5 even.
